services/property-svc/internal/property/api/grpc.go

```diff
@@ -113,3 +113,54 @@ func (s *PropertyGrpcService) GetProperty(ctx context.Context, req *pb.GetProper
 
 	return response, nil
 }
+
+func (s *PropertyGrpcService) UpdateProperty(ctx context.Context, req *pb.UpdatePropertyRequest) (*pb.UpdatePropertyResponse, error) {
+	propertyID, err := uuid.Parse(req.GetId())
+	if err != nil {
+		return nil, err
+	}
+
+	setID, err := hwutil.ParseNullUUID(req.SetId)
+	if err != nil {
+		return nil, err
+	}
+
+	var none *bool
+	var allowFreetext *bool
+	var removeOptions *[]string
+	var upsertOptions *[]models.SelectOption
+
+	switch ftData := req.FieldTypeData.(type) {
+	case *pb.UpdatePropertyRequest_None:
+		none = &ftData.None
```

Comment:
none is not a value, throw everything related to it out (except in the protos ofc)